### ISSUE TITLE
[Feature Request] option or module to prevent automatic reverse

### ISSUE THREAD

**Issue by MauveCloud** [modmuss50/SC2#149]
I made a cart with a reinforced hull, solar engine, coal engine, hardened drill, railer, bridge builder, and drill intelligence.  I want it to drill in one direction for now.  I tracked down some documentation about the module toggler, which seems to work, but that means when it runs out of rails (or finds a gap and is out of bridging material, presumably), it will go back and forth on the existing rails until I break it or stand in front of it to stop it (which luckily doesn't damage me, unlike the Railcraft Tunnel Bore).  I suppose it would also pause at night after it ran out of coal.

Personally, I would prefer to be able to make it sit idle (while facing the same direction) unless I specifically told it to reverse, or gave it what it needed to continue forward.  If there's already a way to do this, I haven't found it.

**Comment by GoryMoon** [comment]
Placing an advanced detector rail will stop the cart until powered, and placing a detector station under it it will stop until the configuration in it is true

**Comment by MauveCloud** [comment]
I can't find documentation of how the detector station works, but wouldn't that require me to set up the detector station ahead of time?

My idea was to have the drill cart stop when it runs out of rails (or bridging material) to place, so I can manually clear the contents of the storage, take things home, then come back with more rails, bridging material, and fuel.  I'm still new with this mod, so maybe that isn't the best way to handle things, but perhaps it will clarify my uncertainty regarding the usefulness of your answer.

**Comment by GoryMoon** [comment]
What most people do and what's intended is that it turn back and unloads at base/where it started and fill up on resources and goes back. There is a setting in the cargo manager to turn around when done or keep going.
You could make it continue and go into a loop or take another way back to the tunnel and use a detector station to stop it if it don't have rails or stop it before the cargo manager or something.

The best info on the detectors that I know are both of theses video https://www.youtube.com/watch?v=14YTLfLRnWE https://www.youtube.com/watch?v=c4abuuoKagE

**Comment by MauveCloud** [comment]
Thanks.  I'm not sure that helped me directly, but it at least gave me a start so I could do some creative-mode experimentation to come up with a good setup.  I'm not sure the detector station is for me, but a module toggler and several cargo managers should be close enough to my underlying goal.  

Alternatively, I could use a seat and advanced control system, now that I've figured out how to decelerate it (the unofficial documentation in the FTB wikis claim that I need to use the sneak key, but that dismounts the cart instead - what I need is the sprint key).

**Comment by GoryMoon** [comment]
Yeah the button had to be changed when it was made the dismount key and the wikis haven't updated that for the newer versions
